src/main/java/se/lth/base/server/mail/CustomEmail.java

```diff
@@ -395,7 +398,7 @@ private String getRatingBody() throws IOException {
         // Action button
         content.put(BUTTON, getButton(WEBSITE_LINK, "Rate Your Trip"));
 
-        return getResource(WEBSITE_LINK + "/drive/" + driveWrap.getDrive().getDriveId(), content);
+        return getResource(WEBSITE_LINK, content);
```

Comment:
This is incorrect, not sure how this got past the tests, should be `return getResource(EMAIL_TEMPLATE, content);`